gcp/shell_startup.sh

```diff
@@ -9,12 +9,16 @@ mkdir -p $HOME/bin
 
 # Install the Lacework CLI if it is not installed
 if [ ! -f "$HOME/bin/lacework" ]; then
+    echo "-> Installing the Lacework CLI..."
     curl https://raw.githubusercontent.com/lacework/go-sdk/master/cli/install.sh | bash -s -- -d $HOME/bin
+else
+    echo "-> The Lacework CLI is already installed!"
+    lacework version
 fi
 
 # Install Terraform if it is not installed
 if [ ! -f "$HOME/bin/terraform" ]; then
-    curl https://releases.hashicorp.com/terraform/0.12.29/terraform_0.12.29_linux_amd64.zip --output $HOME/bin/terraform.zip
+    curl https://releases.hashicorp.com/terraform/0.14.4/terraform_0.14.4_linux_amd64.zip --output $HOME/bin/terraform.zip
```

Comment:
Does Azure always use the latest? if so can we pull down the latest instead of 0.14.4 to keep them consistent. If not might be good have the version set as a variable.

Reply:
I like the idea of installing the latest version, though, I haven't found a way to do that. How would you do it @dmurray-lacework ?

I didn't quite follow the variable idea.